diff -urN applnk-1.9.3.orig/desktop-directories/xfce4-settings.directory applnk-1.9.3/desktop-directories/xfce4-settings.directory
--- applnk-1.9.3.orig/desktop-directories/xfce4-settings.directory	1970-01-01 01:00:00.000000000 +0100
+++ applnk-1.9.3/desktop-directories/xfce4-settings.directory	2004-08-07 17:57:07.772853392 +0200
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=XFCE4
+Comment=XFEC4 settings
+Comment[pl]=Ustawienia XFCE4
+Icon=package_xfce4.png
+# vi: encoding=utf-8
diff -urN applnk-1.9.3.orig/menus/applications.menu applnk-1.9.3/menus/applications.menu
--- applnk-1.9.3.orig/menus/applications.menu	2004-08-07 17:47:57.754468896 +0200
+++ applnk-1.9.3/menus/applications.menu	2004-08-07 17:56:49.939564464 +0200
@@ -725,6 +725,7 @@
 			<Category>PackageManager</Category>
 			</Or>
 			<Not><Category>GNOME</Category></Not>
+			<Not><Category>X-XFCE</Category></Not>
 			</And>
 		</Include>
 	<Menu>
@@ -773,6 +774,16 @@
 		</Include>
 	</Menu>
 	</Menu>
+	<Menu>
+		<Name>XFCE4</Name>
+		<Directory>xfce4-settings.directory</Directory>
+		<Include>
+			<And>
+			<Category>Settings</Category>
+			<Category>X-XFCE</Category>
+			</And>
+		</Include>
+	</Menu>
 	</Menu>
 	
 	<Menu>
