diff -ur applnk-1.9.5.orig/menus/applications.menu applnk-1.9.5/menus/applications.menu
--- applnk-1.9.5.orig/menus/applications.menu	2005-03-25 21:18:04.000000000 +0100
+++ applnk-1.9.5/menus/applications.menu	2005-03-30 15:25:08.000000000 +0200
@@ -823,6 +823,7 @@
 			</Or>
 			<Not><Category>Filesystem</Category></Not>
 			<Not><Category>Monitor</Category></Not>
+			<Not><Category>X-Capplet</Category></Not>
 			</And>
 		</Include>
 		<Menu>
