--- applnk-1.9.4/menus/applications.menu~	Wed Oct 27 12:39:57 2004
+++ applnk-1.9.4/menus/applications.menu	Wed Oct 27 12:39:57 2004
@@ -27,7 +27,14 @@
 		<Name>Administration</Name>
 		<Directory>administration.directory</Directory>
 		<Include>
+			<Or>
+			<!-- to be backward compatible -->
 			<Category>SystemSetup</Category>
+			<And>
+			<Category>System</Category>
+			<Category>X-Administration</Category>
+			</And>
+			</Or>
 		</Include>
 	</Menu>
 	<Move>
