--- applnk-1.9.5/menus/applications.menu.old	2005-06-14 18:38:14.000000000 +0200
+++ applnk-1.9.5/menus/applications.menu	2005-06-14 18:36:45.000000000 +0200
@@ -441,6 +441,7 @@
 			<And>		
 			<Category>AudioVideo</Category>
 			<Not><Category>AudioVideoEditing</Category></Not>
+			<Not><Category>DiscBurning</Category></Not>
 			<Not><Category>Mixer</Category></Not>
 			<Not><Category>Player</Category></Not>
 			<Not><Category>Recorder</Category></Not>
@@ -465,6 +466,16 @@
 			</Or>
 		</Include>
 		<Menu>
+			<Name>CD-RW</Name>
+			<Directory>cd-rw.directory</Directory>
+			<Include>
+				<And>
+				<Category>AudioVideo</Category>
+				<Category>DiscBurning</Category>
+				</And>
+			</Include>
+		</Menu>
+		<Menu>
 			<Name>Editing</Name>
 			<Directory>media-edit.directory</Directory>
 			<Include>
@@ -870,16 +881,6 @@
 			</Include>
 		</Menu>
 		<Menu>
-			<Name>CD-RW</Name>
-			<Directory>cd-rw.directory</Directory>
-			<Include>
-				<And>
-				<Category>Utility</Category>
-				<Category>DiscBurning</Category>
-				</And>
-			</Include>
-		</Menu>
-		<Menu>
 			<Name>Docklets</Name>
 			<Directory>docklets.directory</Directory>
 			<Include>
