diff -ur applnk-1.9.5.orig/desktop-directories/accessibility.directory applnk-1.9.5/desktop-directories/accessibility.directory
--- applnk-1.9.5.orig/desktop-directories/accessibility.directory	2005-02-26 11:28:46.000000000 +0100
+++ applnk-1.9.5/desktop-directories/accessibility.directory	2005-06-04 17:54:09.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Accessibility
+Name[ca]=Accessibilitat
+Name[es]=Accesibilidad
 Name[pl]=Ułatwienia dostępu
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_access.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/administration.directory applnk-1.9.5/desktop-directories/administration.directory
--- applnk-1.9.5.orig/desktop-directories/administration.directory	2005-02-26 11:29:45.000000000 +0100
+++ applnk-1.9.5/desktop-directories/administration.directory	2005-06-04 17:59:09.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Administrations tools
+Name[ca]=Ferramentes d'administració
+Name[es]=Herramientas de administración
 Name[pl]=Narzędzia administracyjne
 Comment=System administrations tools
+Comment[ca]=Ferramentes d'administració del sistema
+Comment[es]=Herramientas de administración del sistema
 Comment[pl]=Narzędzia administracji systemem
 Icon=linuxconf
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/amusements.directory applnk-1.9.5/desktop-directories/amusements.directory
--- applnk-1.9.5.orig/desktop-directories/amusements.directory	2005-02-26 11:48:02.000000000 +0100
+++ applnk-1.9.5/desktop-directories/amusements.directory	2005-06-04 18:07:33.000000000 +0200
@@ -27,6 +27,8 @@
 Name[zh_CN]=消遣娱乐
 Name[zh_TW]=消遣娛樂
 Comment=Amusements
+Comment[ca]=Entreteniments
+Comment[es]=Divertimientos
 Comment[pl]=Aplikacje - zabawki
 Icon=package_toys
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/archiving.directory applnk-1.9.5/desktop-directories/archiving.directory
--- applnk-1.9.5.orig/desktop-directories/archiving.directory	2005-02-26 11:30:29.000000000 +0100
+++ applnk-1.9.5/desktop-directories/archiving.directory	2005-06-04 21:44:05.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Archiving
+Name[ca]=Arxivament
+Name[es]=Archivamiento
 Name[pl]=Archiwizacja
 Comment=Archive managers
+Comment[ca]=Gestió d'arxius
+Comment[es]=Gestión de archivos
 Comment[pl]=Zarządzanie archiwami
 Icon=package_archives.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/art.directory applnk-1.9.5/desktop-directories/art.directory
--- applnk-1.9.5.orig/desktop-directories/art.directory	2005-02-26 11:30:44.000000000 +0100
+++ applnk-1.9.5/desktop-directories/art.directory	2005-06-04 18:06:20.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Art
+Name[ca]=Art
+Name[es]=Arte
 Name[pl]=Plastyka
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_art.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/astronomy.directory applnk-1.9.5/desktop-directories/astronomy.directory
--- applnk-1.9.5.orig/desktop-directories/astronomy.directory	2005-02-26 11:30:55.000000000 +0100
+++ applnk-1.9.5/desktop-directories/astronomy.directory	2005-06-04 18:16:24.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Astronomy
+Name[ca]=Astronomia
+Name[es]=Astronomía
 Name[pl]=Astronomia
 Comment=Scientific applications - Astronomy
+Comment[ca]=Aplicacions científiques - Astronomia
+Comment[es]=Aplicaciones científicas - Astronomía
 Comment[pl]=Aplikacje naukowe - Astronomia
 Icon=package_astronomy.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/calculator.directory applnk-1.9.5/desktop-directories/calculator.directory
--- applnk-1.9.5.orig/desktop-directories/calculator.directory	2005-02-26 11:31:09.000000000 +0100
+++ applnk-1.9.5/desktop-directories/calculator.directory	2005-06-04 18:10:45.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Calculators
+Name[ca]=Calculadores
+Name[es]=Calculadoras
 Name[pl]=Kalkulatory
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_calc.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/cd-rw.directory applnk-1.9.5/desktop-directories/cd-rw.directory
--- applnk-1.9.5.orig/desktop-directories/cd-rw.directory	2005-02-26 11:31:20.000000000 +0100
+++ applnk-1.9.5/desktop-directories/cd-rw.directory	2005-06-04 21:52:09.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=CD-RW Utilities
+Name[ca]=Ferramentes CD-RW
+Name[es]=Herramientas CD-RW
 Name[pl]=Narzędzia CD-RW
 Comment=Utilities for CD recoders
+Comment[ca]=Ferramentes per gravadores CD
+Comment[es]=Herramientas para grabadoras CD
 Comment[pl]=Narzędzia dla nagrywarek płyt CD
 Icon=package_cd-rw.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/chemistry.directory applnk-1.9.5/desktop-directories/chemistry.directory
--- applnk-1.9.5.orig/desktop-directories/chemistry.directory	2005-02-26 11:31:28.000000000 +0100
+++ applnk-1.9.5/desktop-directories/chemistry.directory	2005-06-04 18:16:49.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Chemistry
+Name[ca]=Química
+Name[es]=Química
 Name[pl]=Chemia
 Comment=Scientific applications - Chemistry
+Comment[ca]=Aplicacions científiques - Química
+Comment[es]=Aplicaciones científicas - Química
 Comment[pl]=Aplikacje naukowe - Chemia
 Icon=package_chemistry.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/clock.directory applnk-1.9.5/desktop-directories/clock.directory
--- applnk-1.9.5.orig/desktop-directories/clock.directory	2005-02-26 11:31:36.000000000 +0100
+++ applnk-1.9.5/desktop-directories/clock.directory	2005-06-04 18:18:22.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Clocks
+Name[ca]=Rellotges
+Name[es]=Relojes
 Name[pl]=Zegary
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_clock.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/databases.directory applnk-1.9.5/desktop-directories/databases.directory
--- applnk-1.9.5.orig/desktop-directories/databases.directory	2005-02-26 11:31:55.000000000 +0100
+++ applnk-1.9.5/desktop-directories/databases.directory	2005-06-04 18:19:52.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Databases
+Name[ca]=Bases de dades
+Name[es]=Basas de datos
 Name[pl]=Bazy Danych
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_databases.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/dicts.directory applnk-1.9.5/desktop-directories/dicts.directory
--- applnk-1.9.5.orig/desktop-directories/dicts.directory	2005-02-26 11:32:14.000000000 +0100
+++ applnk-1.9.5/desktop-directories/dicts.directory	2005-06-04 18:21:07.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Dictionaries
+Name[ca]=Diccionaris
+Name[es]=Diccionarios
 Name[pl]=Słowniki
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_dicts.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/docklets.directory applnk-1.9.5/desktop-directories/docklets.directory
--- applnk-1.9.5.orig/desktop-directories/docklets.directory	2005-02-26 11:32:21.000000000 +0100
+++ applnk-1.9.5/desktop-directories/docklets.directory	2005-06-04 18:27:25.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Docklets
+Name[ca]=Docklets
+Name[es]=Docklets
 Name[pl]=Docklety
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_docklets.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/editors.directory applnk-1.9.5/desktop-directories/editors.directory
--- applnk-1.9.5.orig/desktop-directories/editors.directory	2005-02-26 11:32:29.000000000 +0100
+++ applnk-1.9.5/desktop-directories/editors.directory	2005-06-04 18:27:59.000000000 +0200
@@ -8,6 +8,7 @@
 Name[bg]=Редактори
 Name[br]=Aozerien
 Name[bs]=Editori
+Name[ca]=Editors
 Name[cs]=Editory
 Name[da]=Tekstredigering
 Name[de]=Editoren
diff -ur applnk-1.9.5.orig/desktop-directories/edu_music.directory applnk-1.9.5/desktop-directories/edu_music.directory
--- applnk-1.9.5.orig/desktop-directories/edu_music.directory	2005-02-26 11:32:39.000000000 +0100
+++ applnk-1.9.5/desktop-directories/edu_music.directory	2005-06-04 18:29:54.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Music
+Name[ca]=Música
+Name[es]=Música
 Name[pl]=Muzyka
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_music.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/edutainment.directory applnk-1.9.5/desktop-directories/edutainment.directory
--- applnk-1.9.5.orig/desktop-directories/edutainment.directory	2005-02-26 11:32:47.000000000 +0100
+++ applnk-1.9.5/desktop-directories/edutainment.directory	2005-06-04 18:30:21.000000000 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Education
+Name[ca]=Educació
+Name[es]=Educación
 Name[pl]=Edukacja
 Icon=package_edutainment.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/electronics.directory applnk-1.9.5/desktop-directories/electronics.directory
--- applnk-1.9.5.orig/desktop-directories/electronics.directory	2005-02-26 11:33:01.000000000 +0100
+++ applnk-1.9.5/desktop-directories/electronics.directory	2005-06-04 18:31:55.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Electronics
+Name[ca]=Electrònica
+Name[es]=Electrónica
 Name[pl]=Elektronika
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_electro.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/emulators.directory applnk-1.9.5/desktop-directories/emulators.directory
--- applnk-1.9.5.orig/desktop-directories/emulators.directory	2005-02-26 11:34:16.000000000 +0100
+++ applnk-1.9.5/desktop-directories/emulators.directory	2005-06-04 18:33:43.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Emulators
+Name[ca]=Emuladors
+Name[es]=Emuladores
 Name[pl]=Emulatory
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_emu.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/engineering.directory applnk-1.9.5/desktop-directories/engineering.directory
--- applnk-1.9.5.orig/desktop-directories/engineering.directory	2005-02-26 11:48:52.000000000 +0100
+++ applnk-1.9.5/desktop-directories/engineering.directory	2005-06-04 18:37:43.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Engineering
+Name[ca]=Enginyeria
+Name[es]=Ingeniería
 Name[pl]=Inżynieryjne
 Comment=Scientific applications - Engineering
+Comment[ca]=Aplicacions científiques - Enginyeria
+Comment[es]=Aplicaciones científicas - Ingeniería
 Comment[pl]=Aplikacje naukowe - inżynieryjne
 Icon=package_engineering.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/filemanager.directory applnk-1.9.5/desktop-directories/filemanager.directory
--- applnk-1.9.5.orig/desktop-directories/filemanager.directory	2005-02-26 11:34:42.000000000 +0100
+++ applnk-1.9.5/desktop-directories/filemanager.directory	2005-06-04 18:42:31.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=File management
+Name[ca]=Gestors de fitxers
+Name[es]=Gestores de ficheros
 Name[pl]=Menedżery plików
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_fileman.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/filesystem.directory applnk-1.9.5/desktop-directories/filesystem.directory
--- applnk-1.9.5.orig/desktop-directories/filesystem.directory	2005-02-26 11:34:50.000000000 +0100
+++ applnk-1.9.5/desktop-directories/filesystem.directory	2005-06-04 21:49:12.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=File system
+Name[ca]=Sistema d'arxius
+Name[es]=Sistema de archivos
 Name[pl]=System plików
 Comment=Filesystem tools
+Comment[ca]=Ferramentes del sistema d'arxius
+Comment[es]=Herramientas del sistema de archivos
 Comment[pl]=
 Icon=package_filesys.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-action.directory applnk-1.9.5/desktop-directories/games-action.directory
--- applnk-1.9.5.orig/desktop-directories/games-action.directory	2005-02-26 11:35:16.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-action.directory	2005-06-04 19:04:26.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Action
+Name[ca]=Jocs d'acció
+Name[es]=Juegos de acción
 Name[pl]=Gry akcji
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_action.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-adventure.directory applnk-1.9.5/desktop-directories/games-adventure.directory
--- applnk-1.9.5.orig/desktop-directories/games-adventure.directory	2005-02-26 11:35:24.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-adventure.directory	2005-06-04 19:04:46.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Adventure
+Name[ca]=Jocs d'aventura
+Name[es]=Juegos de aventura
 Name[pl]=Przygodowe
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_adventure.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-arcade.directory applnk-1.9.5/desktop-directories/games-arcade.directory
--- applnk-1.9.5.orig/desktop-directories/games-arcade.directory	2005-02-26 11:35:33.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-arcade.directory	2005-06-04 19:01:41.000000000 +0200
@@ -6,10 +6,12 @@
 Name[az]=Əyləncə
 Name[be]=Аркады
 Name[bs]=Arkade
+Name[ca]=Jocs de destresa
 Name[cs]=Arkády
 Name[da]=Arkade
 Name[de]=Arkade
 Name[eo]=Arkado
+Name[es]=Juegos de destreza
 Name[et]=Võitlusmängud
 Name[fo]=Arkaduspøl
 Name[fr]=Jeux d'arcade
diff -ur applnk-1.9.5.orig/desktop-directories/games-blocks.directory applnk-1.9.5/desktop-directories/games-blocks.directory
--- applnk-1.9.5.orig/desktop-directories/games-blocks.directory	2005-02-26 11:35:42.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-blocks.directory	2005-06-04 19:05:43.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Blocks
+Name[ca]=Jocs de blocs
+Name[es]=Juegos de bloques
 Name[pl]=Blokowe
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_block.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-fpp.directory applnk-1.9.5/desktop-directories/games-fpp.directory
--- applnk-1.9.5.orig/desktop-directories/games-fpp.directory	2005-02-26 11:36:10.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-fpp.directory	2005-06-04 19:11:28.000000000 +0200
@@ -2,6 +2,8 @@
 Encoding=UTF-8
 Name=FPP
 Comment="First Person Perspective" games menu
+Comment[ca]=Jocs en "Perspectiva de primera persona"
+Comment[es]=Juegos en "Perspectiva de primera persona"
 Comment[pl]=Menu gier "Widzianych oczami bohatera"
 Icon=package_games_fpp
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-logic.directory applnk-1.9.5/desktop-directories/games-logic.directory
--- applnk-1.9.5.orig/desktop-directories/games-logic.directory	2005-02-26 11:36:18.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-logic.directory	2005-06-04 19:13:26.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Logic
+Name[ca]=Jocs lògics
+Name[es]=Juegos lógicos
 Name[pl]=Logiczne
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_logic.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-racing.directory applnk-1.9.5/desktop-directories/games-racing.directory
--- applnk-1.9.5.orig/desktop-directories/games-racing.directory	2005-02-26 11:36:27.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-racing.directory	2005-06-04 19:17:12.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Racing
+Name[ca]=Jocs de corregudes
+Name[es]=Juegos de carreras
 Name[pl]=Rajdowe
 Comment=Racing games menu
+Comment[ca]=Menú de jocs de corregudes
+Comment[es]=Menú de juegos de carreras
 Comment[pl]=Menu gier rajdowych
 Icon=package_games_racing.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-rpg.directory applnk-1.9.5/desktop-directories/games-rpg.directory
--- applnk-1.9.5.orig/desktop-directories/games-rpg.directory	2005-02-26 11:36:35.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-rpg.directory	2005-06-04 19:28:12.000000000 +0200
@@ -2,6 +2,8 @@
 Encoding=UTF-8
 Name=RPG
 Comment=Role Playing Games menu
+Comment[ca]=Menú de jocs de rol (RPG)
+Comment[es]=Menú de juegos de rol (RPG)
 Comment[pl]=Menu gier fabularnych (RPG)
 Icon=package_games_rpg
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-simulation.directory applnk-1.9.5/desktop-directories/games-simulation.directory
--- applnk-1.9.5.orig/desktop-directories/games-simulation.directory	2005-02-26 11:36:45.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-simulation.directory	2005-06-04 19:30:26.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Simulators
+Name[ca]=Simuladors
+Name[es]=Simuladores
 Name[pl]=Symulatory
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_sim.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-sport.directory applnk-1.9.5/desktop-directories/games-sport.directory
--- applnk-1.9.5.orig/desktop-directories/games-sport.directory	2005-02-26 11:36:55.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-sport.directory	2005-06-04 19:31:09.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Sport
+Name[ca]=Jocs d'esports
+Name[es]=Juegos de deportes
 Name[pl]=Sportowe
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_sport.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/games-strategy.directory applnk-1.9.5/desktop-directories/games-strategy.directory
--- applnk-1.9.5.orig/desktop-directories/games-strategy.directory	2005-02-26 11:37:05.000000000 +0100
+++ applnk-1.9.5/desktop-directories/games-strategy.directory	2005-06-04 19:32:23.000000000 +0200
@@ -7,13 +7,13 @@
 Name[be]=Тактыка і стратэгія
 Name[bg]=Стратегия & Тактика
 Name[bs]=Taktika i strategija
-Name[ca]=Tàctics & d'estratègia
+Name[ca]=Tàctics i d'estratègia
 Name[cs]=Taktické a strategické
 Name[da]=Taktik & strategi
 Name[de]=Taktik & Strategie
 Name[el]=Τακτική & Στρατηγική
 Name[eo]=Taktiko kaj Strategio
-Name[es]=Táctica y estrategia
+Name[es]=Tácticos y de estrategia
 Name[et]=Taktika ja strateegia
 Name[fa]=تاکتیکی و استراتژیک
 Name[fi]=Taktiikka & Strategia
diff -ur applnk-1.9.5.orig/desktop-directories/gnome-settings-accessibility.directory applnk-1.9.5/desktop-directories/gnome-settings-accessibility.directory
--- applnk-1.9.5.orig/desktop-directories/gnome-settings-accessibility.directory	2005-02-26 11:37:40.000000000 +0100
+++ applnk-1.9.5/desktop-directories/gnome-settings-accessibility.directory	2005-06-04 21:52:31.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Accessibility
+Name[ca]=Accessibilitat
+Name[es]=Accesibilidad
 Name[pl]=Dostępność
 Comment=Accessibility Settings
+Comment[ca]=Preferències d'accessibilitat
+Comment[es]=Preferencias de accesibilidad
 Comment[pl]=Ustawienia dostępności
 Icon=package_gnome_accessibility.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/gnome-settings-advanced.directory applnk-1.9.5/desktop-directories/gnome-settings-advanced.directory
--- applnk-1.9.5.orig/desktop-directories/gnome-settings-advanced.directory	2005-02-26 11:37:48.000000000 +0100
+++ applnk-1.9.5/desktop-directories/gnome-settings-advanced.directory	2005-06-04 21:36:44.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Advanced
+Name[ca]=Avançat
+Name[es]=Avanzado
 Name[pl]=Zaawansowane
 Comment=Advanced Settings
+Comment[ca]=Preferències avançades
+Comment[es]=Preferencias avanzadas
 Comment[pl]=Zaawansowane ustawienia
 Icon=package_gnome_advanced.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/gnome-settings.directory applnk-1.9.5/desktop-directories/gnome-settings.directory
--- applnk-1.9.5.orig/desktop-directories/gnome-settings.directory	2005-06-04 17:51:15.000000000 +0200
+++ applnk-1.9.5/desktop-directories/gnome-settings.directory	2005-06-04 19:44:03.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=GNOME settings
+Name[ca]=Configuració de GNOME
+Name[es]=Configuración de GNOME
 Name[pl]=Ustawienia GNOME
 Comment=GNOME settings
+Comment[ca]=Configuració de GNOME
+Comment[es]=Configuración de GNOME
 Comment[pl]=Ustawienia GNOME
 Icon=package_gnome.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/gnome-settings-system.directory applnk-1.9.5/desktop-directories/gnome-settings-system.directory
--- applnk-1.9.5.orig/desktop-directories/gnome-settings-system.directory	2005-02-26 11:37:56.000000000 +0100
+++ applnk-1.9.5/desktop-directories/gnome-settings-system.directory	2005-06-04 19:42:53.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=System Tools
+Name[ca]=Ferramentes del sistema
+Name[es]=Herramientas del sistema
 Name[pl]=Narzędzia systemowe
 Comment=System menu
+Comment[ca]=Menú del sistema
+Comment[es]=Menú del sistema
 Comment[pl]=Menu systemowe
 Icon=package_gnome_system.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/graphics-scan.directory applnk-1.9.5/desktop-directories/graphics-scan.directory
--- applnk-1.9.5.orig/desktop-directories/graphics-scan.directory	2005-02-26 11:38:23.000000000 +0100
+++ applnk-1.9.5/desktop-directories/graphics-scan.directory	2005-06-04 19:52:32.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Scanning
+Name[ca]=Escaneig
+Name[es]=Escaneo
 Name[pl]=Skanowanie
 Comment=Scanning tools
+Comment[ca]=Ferramentes per escanear
+Comment[es]=Herramientas para escanear
 Comment[pl]=Narzędzia do skanowania
 Icon=package_scan.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/graphics-viewers.directory applnk-1.9.5/desktop-directories/graphics-viewers.directory
--- applnk-1.9.5.orig/desktop-directories/graphics-viewers.directory	2005-02-26 11:38:32.000000000 +0100
+++ applnk-1.9.5/desktop-directories/graphics-viewers.directory	2005-06-04 21:50:04.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Viewers
+Name[ca]=Visualitzadors
+Name[es]=Visualizadores
 Name[pl]=Przeglądarki
 Comment=Picture viewers
+Comment[ca]=Visors d'imatges
+Comment[es]=Visores de imágenes
 Comment[pl]=Przeglądarki plików graficznych
 Icon=xeyes
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/help.directory applnk-1.9.5/desktop-directories/help.directory
--- applnk-1.9.5.orig/desktop-directories/help.directory	2005-02-26 11:38:49.000000000 +0100
+++ applnk-1.9.5/desktop-directories/help.directory	2005-06-04 20:12:30.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Help Systems
+Name[ca]=Sistemes d'ajuda
+Name[es]=Sistemas de ayuda
 Name[pl]=Systemy pomocy
 Comment=Menu with tools providing many kinds of help and documentation
+Comment[ca]=Menú amb ferramentes d'ajuda de diferents tipus i documentació
+Comment[es]=Menú con herramientas de ayuda de diferentes tipos y documentación
 Comment[pl]=Menu z narzędziami udostepniajacymi różnego rodzaju pomoc i dokumentację
 Icon=package_help.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/i18n.directory applnk-1.9.5/desktop-directories/i18n.directory
--- applnk-1.9.5.orig/desktop-directories/i18n.directory	2005-02-26 11:38:56.000000000 +0100
+++ applnk-1.9.5/desktop-directories/i18n.directory	2005-06-04 20:13:18.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Internationalization
+Name[ca]=Traduccions
+Name[es]=Traducciones
 Name[pl]=Tłumaczenia
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_i18n.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/languages.directory applnk-1.9.5/desktop-directories/languages.directory
--- applnk-1.9.5.orig/desktop-directories/languages.directory	2005-02-26 11:39:20.000000000 +0100
+++ applnk-1.9.5/desktop-directories/languages.directory	2005-06-04 21:41:27.000000000 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Languages
+Name[ca]=Llenguatges
+Name[es]=Lenguajes
 Name[pl]=Języki
 Icon=package_languages.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/mathematics.directory applnk-1.9.5/desktop-directories/mathematics.directory
--- applnk-1.9.5.orig/desktop-directories/mathematics.directory	2005-02-26 11:39:29.000000000 +0100
+++ applnk-1.9.5/desktop-directories/mathematics.directory	2005-06-04 21:41:39.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Mathematics
+Name[ca]=Matemàtiques
+Name[es]=Matemáticas
 Name[pl]=Matematyka
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_mathematics.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/media-edit.directory applnk-1.9.5/desktop-directories/media-edit.directory
--- applnk-1.9.5.orig/desktop-directories/media-edit.directory	2005-02-26 11:39:37.000000000 +0100
+++ applnk-1.9.5/desktop-directories/media-edit.directory	2005-06-04 20:17:24.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Editing
+Name[ca]=Edició
+Name[es]=Edición
 Name[pl]=Modyfikowanie
 Comment=Media Editing
+Comment[ca]=Edició de medis
+Comment[es]=Edición de medias
 Comment[pl]=Modyfikowanie mediów
 Icon=package_m_edit.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/media-mixers.directory applnk-1.9.5/desktop-directories/media-mixers.directory
--- applnk-1.9.5.orig/desktop-directories/media-mixers.directory	2005-02-26 11:39:50.000000000 +0100
+++ applnk-1.9.5/desktop-directories/media-mixers.directory	2005-06-04 20:22:38.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Mixers
+Name[ca]=Mixers
+Name[es]=Mixers
 Name[pl]=Miksery
 Comment=Mixers
+Comment[ca]=Mixers
+Comment[es]=Mixers
 Comment[pl]=Miksery
 Icon=package_m_mixers.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/media-play.directory applnk-1.9.5/desktop-directories/media-play.directory
--- applnk-1.9.5.orig/desktop-directories/media-play.directory	2005-02-26 11:40:01.000000000 +0100
+++ applnk-1.9.5/desktop-directories/media-play.directory	2005-06-04 20:24:36.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Playback
+Name[ca]=Reproductors
+Name[es]=Reproductores
 Name[pl]=Odtwarzanie
 Comment=Media Playback
+Comment[ca]=Reproducció de medis
+Comment[es]=Reproducción de medias
 Comment[pl]=Odtwarzanie mediów
 Icon=package_m_play.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/media-tuner.directory applnk-1.9.5/desktop-directories/media-tuner.directory
--- applnk-1.9.5.orig/desktop-directories/media-tuner.directory	2005-02-26 11:40:09.000000000 +0100
+++ applnk-1.9.5/desktop-directories/media-tuner.directory	2005-06-04 20:25:07.000000000 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Tuners
+Name[ca]=Tuners
+Name[es]=Tuneres
 Name[pl]=Tunery
 Icon=package_m_tuner.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/misc.directory applnk-1.9.5/desktop-directories/misc.directory
--- applnk-1.9.5.orig/desktop-directories/misc.directory	2005-02-26 11:40:18.000000000 +0100
+++ applnk-1.9.5/desktop-directories/misc.directory	2005-06-04 20:31:23.000000000 +0200
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Miscellaneous
+Name[ca]=Variats
+Name[es]=Variados
 Name[pl]=Różne
 Icon=package_applications.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/monitor.directory applnk-1.9.5/desktop-directories/monitor.directory
--- applnk-1.9.5.orig/desktop-directories/monitor.directory	2005-02-26 11:40:29.000000000 +0100
+++ applnk-1.9.5/desktop-directories/monitor.directory	2005-06-04 20:32:34.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Monitoring
+Name[ca]=Supervisió
+Name[es]=Supervisión
 Name[pl]=Monitorowanie
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_monitor.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/multimedia.directory applnk-1.9.5/desktop-directories/multimedia.directory
--- applnk-1.9.5.orig/desktop-directories/multimedia.directory	2005-02-26 11:40:38.000000000 +0100
+++ applnk-1.9.5/desktop-directories/multimedia.directory	2005-06-04 20:29:09.000000000 +0200
@@ -12,6 +12,7 @@
 Name[el]=Πολυμέσα
 Name[eo]=Sono kaj filmo
 Name[et]=Multimeedia
+Name[es]=Multimedia
 Name[fa]=چندرسانه‌ای
 Name[fo]=Fleirmiðla
 Name[fr]=Multimédia
diff -ur applnk-1.9.5.orig/desktop-directories/network-communications.directory applnk-1.9.5/desktop-directories/network-communications.directory
--- applnk-1.9.5.orig/desktop-directories/network-communications.directory	2005-02-26 11:40:46.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-communications.directory	2005-06-04 20:35:01.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Communications
+Name[ca]=Comunicació
+Name[es]=Comunicación
 Name[pl]=Komunikacja
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_communications.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network-dialup.directory applnk-1.9.5/desktop-directories/network-dialup.directory
--- applnk-1.9.5.orig/desktop-directories/network-dialup.directory	2005-02-26 11:40:54.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-dialup.directory	2005-06-04 21:37:07.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Connections
+Name[ca]=Connexions
+Name[es]=Conexiones
 Name[pl]=Połączenia
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_dialup.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network.directory applnk-1.9.5/desktop-directories/network.directory
--- applnk-1.9.5.orig/desktop-directories/network.directory	2005-02-26 11:48:02.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network.directory	2005-06-04 21:53:13.000000000 +0200
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=Network
 Name[bg]=Интернет
-Name[ca]=Internet
+Name[ca]=Xarxa
 Name[cs]=Síť
 Name[da]=Internet
 Name[de]=Netzwerk
@@ -28,7 +28,7 @@
 Name[zh_TW]=網際網路
 Comment=Programs for Internet and networks
 Comment[bg]=Програми за Интернет и Мрежи
-Comment[ca]=Ferramentres per treballar amb La Xarxa
+Comment[ca]=Ferramentes per xarxes i Internet
 Comment[da]=Progammer for Internet og netværk
 Comment[de]=Programme für Netzwerk oder Internet
 Comment[es]=Programas para redes e Internet
diff -ur applnk-1.9.5.orig/desktop-directories/network-ftp.directory applnk-1.9.5/desktop-directories/network-ftp.directory
--- applnk-1.9.5.orig/desktop-directories/network-ftp.directory	2005-02-26 11:41:04.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-ftp.directory	2005-06-04 21:34:41.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=File Transfer
+Name[ca]=Transmissió d'arxius
+Name[es]=Transmisión de archivos
 Name[pl]=Transfer plików
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_network_ftp.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network-mail.directory applnk-1.9.5/desktop-directories/network-mail.directory
--- applnk-1.9.5.orig/desktop-directories/network-mail.directory	2005-02-26 11:41:11.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-mail.directory	2005-06-04 20:39:26.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Mail
+Name[ca]=Correu
+Name[es]=Correo
 Name[pl]=Poczta
 Comment=E-mail programs
+Comment[ca]=Programes de correu
+Comment[es]=Programas de correo
 Comment[pl]=Programy pocztowe
 Icon=package_network_mail.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network-news.directory applnk-1.9.5/desktop-directories/network-news.directory
--- applnk-1.9.5.orig/desktop-directories/network-news.directory	2005-02-26 11:41:17.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-news.directory	2005-06-04 21:53:26.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=News
-Comment=Usenet and newstickers
+Name[ca]=Notícies
+Name[es]=Noticias
 Name[pl]=Wiadomości
+Comment=Usenet and newstickers
+Comment[ca]=Grups de discussió i notícies
+Comment[es]=Grupos de discusión i noticias
 Comment[pl]=Grupy dyskusyjne i wiadomości
 Icon=package_network_news.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network-remote.directory applnk-1.9.5/desktop-directories/network-remote.directory
--- applnk-1.9.5.orig/desktop-directories/network-remote.directory	2005-02-26 11:41:25.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-remote.directory	2005-06-04 20:42:56.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Remote Access
+Name[ca]=Accés remot
+Name[es]=Acceso remoto
 Name[pl]=Zdalny dostęp
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_remote.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/network-www.directory applnk-1.9.5/desktop-directories/network-www.directory
--- applnk-1.9.5.orig/desktop-directories/network-www.directory	2005-02-26 11:41:34.000000000 +0100
+++ applnk-1.9.5/desktop-directories/network-www.directory	2005-06-04 20:46:21.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=WWW browsers
+Name[ca]=Navegadors WWW
+Name[es]=Navegadores WWW
 Name[pl]=Przeglądarki WWW
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_network_www.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/numerics.directory applnk-1.9.5/desktop-directories/numerics.directory
--- applnk-1.9.5.orig/desktop-directories/numerics.directory	2005-02-26 11:42:13.000000000 +0100
+++ applnk-1.9.5/desktop-directories/numerics.directory	2005-06-04 20:49:53.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Numerics
+Name[ca]=Numèrics
+Name[es]=Numéricos
 Name[pl]=Numeryka
 Comment=Scientific applications - Numerics
+Comment[ca]=Aplicacions científiques - Numèrics
+Comment[es]=Aplicaciones científicas - Numéricos
 Comment[pl]=Aplikacje naukowe - Numeryka
 Icon=package_numerics.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/pda.directory applnk-1.9.5/desktop-directories/pda.directory
--- applnk-1.9.5.orig/desktop-directories/pda.directory	2005-02-26 11:42:30.000000000 +0100
+++ applnk-1.9.5/desktop-directories/pda.directory	2005-06-04 20:50:56.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=PDA's
+Name[ca]=PDAs
+Name[es]=PDAs
 Name[pl]=Palmtopy
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_pda.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/physics.directory applnk-1.9.5/desktop-directories/physics.directory
--- applnk-1.9.5.orig/desktop-directories/physics.directory	2005-02-26 11:42:42.000000000 +0100
+++ applnk-1.9.5/desktop-directories/physics.directory	2005-06-04 20:52:04.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Physics
+Name[ca]=Física
+Name[es]=Física
 Name[pl]=Fizyka
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_physics.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/pims.directory applnk-1.9.5/desktop-directories/pims.directory
--- applnk-1.9.5.orig/desktop-directories/pims.directory	2005-02-26 11:42:49.000000000 +0100
+++ applnk-1.9.5/desktop-directories/pims.directory	2005-06-04 20:52:52.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Personal organizers
+Name[ca]=Organitzadors
+Name[es]=Organizadores
 Name[pl]=Organizery
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_pims.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/plotting.directory applnk-1.9.5/desktop-directories/plotting.directory
--- applnk-1.9.5.orig/desktop-directories/plotting.directory	2005-02-26 11:42:59.000000000 +0100
+++ applnk-1.9.5/desktop-directories/plotting.directory	2005-06-04 20:54:54.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Plotting
+Name[ca]=Traçats
+Name[es]=Trazados
 Name[pl]=Wykresy
 Comment=Scientific applications - Plotting
+Comment[ca]=Aplicacions científiques - Traçats
+Comment[es]=Aplicaciones científicas - Trazados
 Comment[pl]=Aplikacje naukowe - Wykresy
 Icon=package_plotting.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/presentations.directory applnk-1.9.5/desktop-directories/presentations.directory
--- applnk-1.9.5.orig/desktop-directories/presentations.directory	2005-02-26 11:43:06.000000000 +0100
+++ applnk-1.9.5/desktop-directories/presentations.directory	2005-06-04 20:55:36.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Presentations
+Name[ca]=Presentacions
+Name[es]=Presentaciones
 Name[pl]=Prezentacje
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_presentations.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/project.directory applnk-1.9.5/desktop-directories/project.directory
--- applnk-1.9.5.orig/desktop-directories/project.directory	2005-02-26 11:43:14.000000000 +0100
+++ applnk-1.9.5/desktop-directories/project.directory	2005-06-04 20:57:38.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Projects Organisation
+Name[ca]=Organització de projectes
+Name[es]=Organización de proyectos
 Name[pl]=Organizacja projektów
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_project.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/scientific.directory applnk-1.9.5/desktop-directories/scientific.directory
--- applnk-1.9.5.orig/desktop-directories/scientific.directory	2005-02-26 11:43:24.000000000 +0100
+++ applnk-1.9.5/desktop-directories/scientific.directory	2005-06-04 21:01:52.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Science
+Name[ca]=Ciència
+Name[es]=Ciencia
 Name[pl]=Nauki ścisłe
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_scientific.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/spreadsheets.directory applnk-1.9.5/desktop-directories/spreadsheets.directory
--- applnk-1.9.5.orig/desktop-directories/spreadsheets.directory	2005-02-26 11:43:42.000000000 +0100
+++ applnk-1.9.5/desktop-directories/spreadsheets.directory	2005-06-04 21:12:01.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Spreadsheets
+Name[ca]=Fulls de càlcul
+Name[es]=Hojas de cálculo
 Name[pl]=Arkusze kalkulacyjne
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_spreadsheet.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/teaching.directory applnk-1.9.5/desktop-directories/teaching.directory
--- applnk-1.9.5.orig/desktop-directories/teaching.directory	2005-02-26 11:44:04.000000000 +0100
+++ applnk-1.9.5/desktop-directories/teaching.directory	2005-06-04 21:14:09.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Teaching
+Name[ca]=Ensenyament
+Name[es]=Enseñanza
 Name[pl]=Nauczanie
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_teach.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/terminals.directory applnk-1.9.5/desktop-directories/terminals.directory
--- applnk-1.9.5.orig/desktop-directories/terminals.directory	2005-02-26 11:44:12.000000000 +0100
+++ applnk-1.9.5/desktop-directories/terminals.directory	2005-06-04 21:16:00.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Terminals
+Name[ca]=Terminals
+Name[es]=Terminales
 Name[pl]=Terminale
 Comment=Terminal applications
+Comment[ca]=Aplicacions terminals
+Comment[es]=Aplicaciones terminales
 Comment[pl]=Aplikacje terminalowe
 Icon=package_terminals.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/webdesign.directory applnk-1.9.5/desktop-directories/webdesign.directory
--- applnk-1.9.5.orig/desktop-directories/webdesign.directory	2005-02-26 11:44:27.000000000 +0100
+++ applnk-1.9.5/desktop-directories/webdesign.directory	2005-06-04 21:17:44.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Web Design
+Name[ca]=Disseny de pàgines WWW
+Name[es]=Diseño de paginas WWW
 Name[pl]=Projektowanie stron WWW
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_webdesign.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/wordprocessors.directory applnk-1.9.5/desktop-directories/wordprocessors.directory
--- applnk-1.9.5.orig/desktop-directories/wordprocessors.directory	2005-02-26 11:44:38.000000000 +0100
+++ applnk-1.9.5/desktop-directories/wordprocessors.directory	2005-06-04 21:36:06.000000000 +0200
@@ -1,8 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Wordprocessors
+Name[ca]=Processadors de text
+Name[es]=Procesadores de texto
 Name[pl]=Procesory tekstu
 Comment=
+Comment[ca]=
+Comment[es]=
 Comment[pl]=
 Icon=package_wordprocessing.png
 Type=Directory
diff -ur applnk-1.9.5.orig/desktop-directories/xfce4-settings.directory applnk-1.9.5/desktop-directories/xfce4-settings.directory
--- applnk-1.9.5.orig/desktop-directories/xfce4-settings.directory	2005-02-26 11:45:10.000000000 +0100
+++ applnk-1.9.5/desktop-directories/xfce4-settings.directory	2005-06-04 21:23:15.000000000 +0200
@@ -2,6 +2,8 @@
 Encoding=UTF-8
 Name=XFCE4
 Comment=XFEC4 settings
+Comment[ca]=Preferències de XFCE4
+Comment[es]=Preferencias de XFCE4
 Comment[pl]=Ustawienia XFCE4
 Icon=package_xfce4.png
 Type=Directory
