diff -aurN applnk-1.9.5.orig/desktop-directories/gnome-settings.directory applnk-1.9.5/desktop-directories/gnome-settings.directory
--- applnk-1.9.5.orig/desktop-directories/gnome-settings.directory	2005-02-26 11:48:02.000000000 +0100
+++ applnk-1.9.5/desktop-directories/gnome-settings.directory	2005-05-20 22:37:10.000000000 +0200
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=GNOME
+Name=GNOME settings
+Name[pl]=Ustawienia GNOME
 Comment=GNOME settings
 Comment[pl]=Ustawienia GNOME
 Icon=package_gnome.png
diff -aurN applnk-1.9.5.orig/menus/applications.menu applnk-1.9.5/menus/applications.menu
--- applnk-1.9.5.orig/menus/applications.menu	2005-03-25 21:18:04.000000000 +0100
+++ applnk-1.9.5/menus/applications.menu	2005-05-20 22:37:25.000000000 +0200
@@ -749,6 +749,7 @@
 			</Or>
 			<Not><Category>X-XFCE</Category></Not>
 			<Not><Category>X-Capplet</Category></Not>
+			<Not><Filename>gnomecc.desktop</Filename></Not>
 			</And>
 		</Include>
 		<Menu>
@@ -761,56 +762,6 @@
 				</And>
 			</Include>
 		</Menu>
-		<Menu>
-			<Name>GNOME</Name>
-			<Directory>gnome-settings.directory</Directory>
-			<Include>
-				<And>
-				<Category>Settings</Category>
-				<Category>GNOME</Category>
-				<Category>X-Capplet</Category>
-				<Not><Category>AdvancedSettings</Category></Not>
-				<Not><Category>System</Category></Not>
-				<Not><Category>Accessibility</Category></Not>
-				</And>
-			</Include>
-			<Menu>
-				<Name>Advanced</Name>
-				<Directory>gnome-settings-advanced.directory</Directory>
-				<Include>
-					<And>
-					<Category>Settings</Category>
-					<Category>GNOME</Category>
-					<Category>X-Capplet</Category>
-					<Category>AdvancedSettings</Category>
-					</And>
-				</Include>
-			</Menu>
-			<Menu>
-				<Name>System</Name>
-				<Directory>gnome-settings-system.directory</Directory>
-				<Include>
-					<And>
-					<Category>Settings</Category>
-					<Category>GNOME</Category>
-					<Category>X-Capplet</Category>
-					<Category>System</Category>
-					</And>
-				</Include>
-			</Menu>
-			<Menu>
-				<Name>Accessibility</Name>
-				<Directory>gnome-settings-accessibility.directory</Directory>
-				<Include>
-					<And>
-					<Category>Settings</Category>
-					<Category>GNOME</Category>
-					<Category>X-Capplet</Category>
-					<Category>Accessibility</Category>
-					</And>
-				</Include>
-			</Menu>
-		</Menu>
 	</Menu>
 	<Menu>
 		<Name>System</Name>
@@ -982,4 +933,13 @@
 	<DefaultMergeDirs/>
 	<!-- For user menu modified with kmenuedit -->
 	<MergeFile>applications-kmenuedit.menu</MergeFile>
+	
+	<!-- Duplicated entries (already in 'desktop' menu) -->
+	<Exclude>
+		<Filename>gnome-about.desktop</Filename>
+		<Filename>nautilus.desktop</Filename>
+		<Filename>nautilus-computer.desktop</Filename>
+		<Filename>nautilus-home.desktop</Filename>
+		<Filename>network-scheme.desktop</Filename>
+	</Exclude>
 </Menu>
