diff -ur applnk-2.2.0/menus/applnk-applications.menu applnk-2.2.0-xfce/menus/applnk-applications.menu
--- applnk-2.2.0/menus/applnk-applications.menu	2011-02-22 11:20:34.000000000 +0100
+++ applnk-2.2.0-xfce/menus/applnk-applications.menu	2012-05-18 08:35:37.000000000 +0200
@@ -26,7 +26,7 @@
 	</Include>
 
 	<Layout>
-		<Filename>xfrun4.desktop</Filename>
+		<Filename>xfce4-run.desktop</Filename>
 		<Separator/>
 		<Filename>exo-terminal-emulator.desktop</Filename>
 		<Filename>exo-file-manager.desktop</Filename>
@@ -38,7 +38,6 @@
 		<Separator/>
 		<Menuname>Settings</Menuname>
 		<Separator/>
-		<Filename>xfhelp4.desktop</Filename>
 		<Filename>xfce4-about.desktop</Filename>
 		<Filename>xfce4-session-logout.desktop</Filename>
 	</Layout>
@@ -621,6 +620,7 @@
 			<Category>PackageManager</Category>
 			<Category>Printing</Category>
 			</Or>
+			<Not><Category>XFCE</Category></Not>
 			<Not><Category>X-XFCE</Category></Not>
 			<Not><Category>GNOME</Category></Not>
 			</And>
@@ -631,15 +631,71 @@
 			<Include>
 				<And>
 				<Category>Settings</Category>
+				<Or>
 				<Category>X-XFCE</Category>
+				<Category>XFCE</Category>
+				</Or>
+				<Not>
+				<Or>
+				<Category>X-XFCE-PersonalSettings</Category>
+				<Category>X-XFCE-HardwareSettings</Category>
+				<Category>X-XFCE-SystemSettings</Category>
+				</Or>
+				</Not>
 				</And>
 			</Include>
 			<Layout>
-				<Filename>xfce-settings-manager.desktop</Filename>
-				<Separator/>
-				<Merge type="menus"/>
-				<Merge type="files"/>
+				<Menuname>Personal</Menuname>
+				<Menuname>Hardware</Menuname>
+				<Menuname>System</Menuname>
+				<Merge type="all"/>
 			</Layout>
+			<Menu>
+				<Name>Personal</Name>
+				<Directory>xfce-personal.directory</Directory>
+				<Include>
+					<And>
+					<Category>Settings</Category>
+					<Or>
+					<Category>X-XFCE</Category>
+					<Category>XFCE</Category>
+					</Or>
+					<Category>X-XFCE-SettingsDialog</Category>
+					<Category>X-XFCE-PersonalSettings</Category>
+					</And>
+					<Filename>xscreensaver-properties.desktop</Filename>
+				</Include>
+			</Menu>
+			<Menu>
+				<Name>Hardware</Name>
+				<Directory>xfce-hardware.directory</Directory>
+				<Include>
+					<And>
+					<Category>Settings</Category>
+					<Or>
+					<Category>X-XFCE</Category>
+					<Category>XFCE</Category>
+					</Or>
+					<Category>X-XFCE-SettingsDialog</Category>
+					<Category>X-XFCE-HardwareSettings</Category>
+					</And>
+				</Include>
+			</Menu>
+			<Menu>
+				<Name>System</Name>
+				<Directory>xfce-system.directory</Directory>
+				<Include>
+					<And>
+					<Category>Settings</Category>
+					<Or>
+					<Category>X-XFCE</Category>
+					<Category>XFCE</Category>
+					</Or>
+					<Category>X-XFCE-SettingsDialog</Category>
+					<Category>X-XFCE-SystemSettings</Category>
+					</And>
+				</Include>
+			</Menu>
 		</Menu>
 		<Menu>
 			<Name>GNOME</Name>
